olwebsite-app/src/components/Toolbar/__tests__/HamburgerToolbar.test.jsx

```diff
@@ -1,32 +1,35 @@
 import React from 'react';
 
-import { render, fireEvent, getByLabelText } from '@testing-library/react';
 import HamburgerToolbar from '../HamburgerToolbar';
-import {  BrowserRouter as Router } from "react-router-dom";
+import { renderWithRouter, getComponentFromDOM, clickEventByLabelText} from "../../../utils/test-utils";
```

Comment:
You can import from test-utils like it's a package. Are you using a relative import as a preference?